packages/protocol/genesis/GenerateGenesis.g.sol

```diff
@@ -77,70 +74,62 @@ contract TestGenerateGenesis is Test, AddressResolver {
     }
 
     function testAddressManager() public {
-        AddressManager addressManager =
+        AddressManager addressManagerProxy =
```

Comment:
We should simplify the names buy removing "Proxy", "Singleton"... as all contracts are behnind proxy.